src/index.js

```diff
@@ -69,14 +70,17 @@ module.exports = class BlockService {
       return callback(new Error('Invalid batch of multihashes'))
     }
 
-    var results = {}
+    if (this.isOnline()) {
+      this._bitswap.getBlocks(multihashes, (results) => {
+        callback(null, results)
+      })
+      return
+    }
```

Comment:
Should it be the other way, first check if we have it locally, then check on bitswap?